### PR TITLE
Slideshow implementation

### PR DESCRIPTION
**Pull request by jtfwong**
Unsure if working as my laptop is very slow and is very difficult to test on. Slideshow button located in view problem. Design shows a blank activity for the slideshow, but if it works, should display photos in the white area. User navigates through photos by swiping. Still needs to be tested and probably bug fixed.

screen shot 2018-11-29 at 1 36 19 am
screen shot 2018-11-29 at 1 36 43 am

### CODE REVIEW

**Comment by itstc** [comment]
Hey the code looks fine to me, is it possible to have a screenshot up so I can get a better overview of the flow?

File: app/src/main/java/com/team7/cmput301/android/theirisproject/activity/ProblemSlideshowActivity.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    private RecordController controller;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_problem_slideshow);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        controller = new RecordController(getIntent());
```
Comment (itstc) [inline app/src/main/java/com/team7/cmput301/android/theirisproject/activity/ProblemSlideshowActivity.java]: Slideshow should have a list of records so maybe RecordListController is more suitable in this situation

Comment (jtfwong) [reply app/src/main/java/com/team7/cmput301/android/theirisproject/activity/ProblemSlideshowActivity.java]: Suggestion has been implemented and comments have been added

File: app/src/main/java/com/team7/cmput301/android/theirisproject/task/GetRecordPhotosTask.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+/**
+ * GetRecordPhotosTask queries the record photos associated to the problem
+ *
+ * @author jtfwong
+ * */
+public class GetRecordPhotosTask extends AsyncTask<String, List<RecordPhoto>, Void> {
+
+    private Callback cb;
+    private String queryString;
```
Comment (itstc) [inline app/src/main/java/com/team7/cmput301/android/theirisproject/task/GetRecordPhotosTask.java]: set this to "{"query": {	"term": {"recordId": "%s" }}}"

File: app/src/main/java/com/team7/cmput301/android/theirisproject/task/GetRecordPhotosTask.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    @Override
+    protected Void doInBackground(String... strings) {
+        new GetRecordListTask(new Callback<SearchResult>() {
+            @Override
+            public void onComplete(SearchResult res) {
+                List<RecordPhoto> recordPhotos = new ArrayList<>();
+                List<Record> recordList = res.getSourceAsObjectList(Record.class, true);
+                for (Record r: recordList) {
+                    Search search = new Search.Builder(queryString).addIndex(IrisProjectApplication.INDEX).addType("recordphoto").build();
```
Comment (itstc) [inline app/src/main/java/com/team7/cmput301/android/theirisproject/task/GetRecordPhotosTask.java]: replace Search.Builder(queryString)... to Search.Builder(String.format(queryString, r.getId()))...

**Pull request closed without merging**